api/tests/src/db/models/factories.py

```diff
@@ -1858,7 +1858,7 @@ class Meta:
     user = factory.SubFactory(UserFactory)
     user_id = factory.LazyAttribute(lambda s: s.user.user_id)
 
-    external_user_type_id = factory.fuzzy.FuzzyChoice(ExternalUserType)
+    external_user_type = factory.fuzzy.FuzzyChoice(ExternalUserType)
```

Comment:
Was this required as part of the PR? Do tests fail if this is `external_user_type_id`

Reply:
yup they do fail. This fixes it.